jenkins/compy_intel_mam4xx_compare_performance.sh

```diff
@@ -18,10 +18,10 @@ main() {
     resolution=$1
 
     #set timestep
-    timestep=5
+    timestep=Ld5
 
     #set data destination
-    data_dest="/qfs/projects/eagles/litz372/performance_data"
+    data_dest="/qfs/projects/eagles/litz372/performance_data/${timestep}"
```

Comment:
Could you please make these comments more descriptive?
Instead of `#set data destination`, we can say `#Destination of the file for all the past and current data"
Instead of `#set shared data destination`, we can say `#Path where the latest time series plot is saved"